tools/changelog/changelog/entry.go

```diff
@@ -0,0 +1,60 @@
+package changelog
+
+import (
+	"fmt"
+	"strings"
+)
+
+type Changelog struct {
+	// One or more changes
+	Changes []*Change `yaml:"changes,omitempty"`
+}
+
+func (cl *Changelog) Merge(other Changelog) Changelog {
+	var changes []*Change
```

Comment:
perf nit, not that it matters for a program like this:
```suggestion
	var changes = make([]*Change, 0, len(cl)+len(other))
```